src/components/CaptureDetailDialog.js

```diff
@@ -100,7 +103,11 @@ function CaptureDetailDialog(props) {
   const classes = useStyles();
 
   useEffect(() => {
-    cdContext.getCaptureDetail(capture?.id);
+    // prevent request if capture is empty or undefined because it will fail without a valid id
+    // but if we have capture info, do we need to make the request?
+    if (capture?.id) {
+      cdContext.getCaptureDetail(capture);
+    }
   }, [capture]);
```

Comment:
I think Dialogs in Material UI might be designed to receive props.  If we pass props then we are already getting all the info we need.  If we don't pass any props we wouldn't have the id to make the request.  We could pass the id for the request through a route, possibly, but I think we'd have to add in react router or something else to support the detail view.

Reply:
As discussed, it's best if the Capture Detail dialog can be populated when passed just a reference ID, raw capture ID or processed capture ID. The dialog (or corresponding context) can then build up a complete picture of the capture, its status and associated wallet registration, device info etc.

There's a case to move this aggregation to a Capture Query API, as a lot of the data has been requested in the Captures list as well.